### PR TITLE
Move Wrapping<T> ui tests into library

### PR DESCRIPTION
**Pull request by workingjubilee**
Part of #76268
r? @matklad

### CODE REVIEW

File: library/core/tests/num/wrapping.rs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            wrapping_operation!(zero, zero >> 1usize);
+            wrapping_assignment!(zero, zero >>= 1usize);
+            wrapping_operation!(zero, -zero);
+            wrapping_operation!(max, !min);
+        }
+    };
+}
+
+#[cfg(tests)]
+mod tests {
```
Comment (lzutao) [inline library/core/tests/num/wrapping.rs:64]: We don't need the mod and cfg.

**Comment by matklad** [comment]
r=me with [this comment](https://github.com/rust-lang/rust/pull/76745#discussion_r489143459) addressed 

@bors delegate=workingjubilee

File: library/core/tests/num/wrapping.rs (hunk 1 above)
Comment (workingjubilee) [reply library/core/tests/num/wrapping.rs:64]: Ah, thank you!

**Comment by matklad** [comment]
@bors r+ rollup

Thanks!

**Pull request merged**